pkg/networkmanager/manifest.json.in

```diff
@@ -24,7 +24,7 @@
                     "matches": ["network", "interface", "bridge", "vlan", "bond", "team", "port", "mac", "ipv4", "ipv6"]
                 },
                 {
-                    "matches": ["firewall", "zone", "tcp", "udp"],
+                    "matches": ["firewall", "firewalld" "zone", "tcp", "udp"],
```

Comment:
/me tosses you a comma :grin:

Reply:
[toss a comma to your Andreas](https://www.youtube.com/watch?v=hqbS7O9qIXE) :smile:

Reply:
Doh. Fixed. Thanks!